osu.Game/Rulesets/Judgements/DrawableJudgement.cs

```diff
@@ -65,22 +65,24 @@ protected override void LoadComplete()
 
             this.FadeInFromZero(100, Easing.OutQuint);
 
+            var origScale = Scale;
+
             switch (Result.Type)
             {
                 case HitResult.None:
                     break;
                 case HitResult.Miss:
-                    this.ScaleTo(1.6f);
-                    this.ScaleTo(1, 100, Easing.In);
+                    this.ScaleTo(origScale * 1.6f);
```

Comment:
probably shouldn't be adjusting ones' own scale, but i'll let this slide this time

Reply:
What would be better in the future, just wondering